src/main/java/com/aws/greengrass/clientdevices/auth/metrics/MetricsEmitter.java

```diff
@@ -37,8 +43,10 @@ public void start(int periodicAggregateIntervalSec) {
         synchronized (emitMetricsLock) {
             // Cancel previously running task
             stop();
-            future = ses.scheduleWithFixedDelay(metrics::emitMetrics, 0, periodicAggregateIntervalSec,
-                    TimeUnit.SECONDS);
+            PutComponentMetricRequest request = new PutComponentMetricRequest();
+            request.setMetrics(metrics.collectMetrics());
+            future = ses.scheduleWithFixedDelay((Runnable) putComponentMetricOperationHandler.handleRequest(request),
```

Comment:
this is just going to push the same metrics over and over again. You need to collect the metrics and create a new request object every time this scheduled task executes.